components/ui/AppPageHeaderFixed.vue

```diff
@@ -1,39 +1,57 @@
 <template>
   <header class="app-page-header-fixed">
-    <div class="app-page-header-fixed__container bx--row">
+    <div
```

Comment:
Could you try something like this?:

```
    <div class="app-page-header-fixed__container"
     :class="{ 'bx--grid' : !oldContainer,
                     'app-page-header-fixed__old-container: oldContainer}">
      <div class="bx--row">
        <h1 class="bx--col-md-6 bx--col-lg-10"
         :class="{ 'bx--col-sm bx--col-max-8': !oldContainer,
                         'app-page-header-fixed__headline': oldContainer}"
          <slot />
        </h1>
      </div>
    </div>
```
I prefer to have this instead of `v-if v-else` because the structure of the components is the same, the only thing that changes are the classes

Note that if you apply this version, you can delete the `width: 100%;` on L46

Reply:
Nice idea! Will do!